src/components/Autocomplete/Autocomplete.jsx

```diff
@@ -58,6 +58,34 @@ class TetheredSelect extends Select {
   }
 }
 
+/**
+ * This is a custom version of the react-select component,
+ * specifically when creatable options are allowed,
+ * that will allow the options menu to overflow and be visible
+ * in containers that have overflow: hidden, scroll, etc.
+ *
+ * https://github.com/JedWatson/react-select/issues/810#issuecomment-284573308
+ */
+const TetheredCreatable = props => {
+  return (
+    <TetherComponent
```

Comment:
I feel it may be worth pulling out this `TetherComponent` (with its props) and creating a wrapper component with it. That way we can share most of the code used in lines 40-56, and just pass different `children` to the wrapper based on the situation

Reply:
Good suggestion. I will make it so.